utils/users/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "users",
+  "version": "1.0.0",
+  "description": "## Static list of SPI users to collect keycloak information and insert into biohub db",
+  "main": "index.js",
+  "scripts": {
```

Comment:
Should we add a description in the README of what `npm run users` and `npm run insert` do?